.flowconfig

```diff
@@ -5,16 +5,14 @@
 <PROJECT_ROOT>/packages/gestalt-codemods/.*/__testfixtures__
 <PROJECT_ROOT>/scripts/templates/
 <PROJECT_ROOT>/node_modules/react-docgen/node_modules
+<PROJECT_ROOT>/node_modules/@rollup/plugin-typescript/node_modules
 
 [options]
 exact_by_default=true
 include_warnings=true
 react.runtime=automatic
 sharedmemory.hash_table_pow=21
 
-[libs]
```

Comment:
eye candy commit

Reply:
@AlbertCarreras Actually, removing this doesn't mean that we don't use `flow-typed` anymore. In fact, we are still using because small part of the codebase (like eslint linter rules and codemod) still use Flow. It's just that the default value for libs is already `flow-typed` (see docs https://flow.org/en/docs/config/libs/), so we shouldn't need to declare it here.

With that said, I did get to remove majority of the flow-typed files, so it's a big win (even though this line doesn't actually covey that).